src/functions-reference/matrix_operations.Rmd

```diff
@@ -2230,6 +2230,17 @@ The matrix with the complex-valued (column) eigenvectors of the matrix `A` in th
 same order as returned by the function `eigenvalues`
 `r since("2.30")`
 
+
+<!-- tuple(complex_matrix, complex_vector); eigendecompose; (matrix A); -->
```

Comment:
I think this needs to go on the complex_matrix_operations section if any complex component pollutes like for Schur return for real inputs.

Reply:
I don't know if I'm super happy with the way we have these pages divided up, the reasoning behind putting the complex-return eigendecomposition functions here is that the `_sym` functions already were, and if someone is looking at those but wants a more general function it seems odd to have on a different page.

Reply:
OK.  I agree that it's a bit of a mess.  I was the one who created the original taxonomy here and am totally OK with changing it if you have a better suggestion.  That doesn't need to happen in this PR, of course.